namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-staging/resources/rds.tf

```diff
@@ -57,7 +77,11 @@ resource "kubernetes_secret" "rds-snapshot" {
   data = {
     access_key_id     = module.rds-instance.access_key_id
     secret_access_key = module.rds-instance.secret_access_key
-    url               = "postgres://${module.rds-instance.database_username}:${module.rds-instance.database_password}@${module.rds-instance.rds_instance_endpoint}/${module.rds-instance.database_name}"
+    username          = module.rds-instance.database_username
+    password          = module.rds-instance.database_password
+    endpoint          = module.rds-snapshot.rds_instance_endpoint
+    database_name     = module.rds-snapshot.database_name
+    url               = "postgres://${username}:${password}@${endpoint}/${database_name}"
```

Comment:
It's not possible to use `username`, `passwords, `endpoint`, etc as variable in your `url` declaration.
Please build the `url` secret by concatenating all the  `module.rds....`